views/chitter.erb

```diff
@@ -0,0 +1 @@
+<%= @peep %>
```

Comment:
Could this have a link to the chitter/new page so that multiple peeps can be added?